Makefile.am

```diff
@@ -2777,7 +2777,7 @@ hlint: $(HS_BUILT_SRCS) src/lint-hints.hs
 	@test -n "$(HLINT)" || { echo 'hlint' not found during configure; exit 1; }
 	@rm -f doc/hs-lint.html
 	if tty -s; then C="-c"; else C=""; fi; \
-	$(HLINT) --utf8 --report=doc/hs-lint.html --cross $$C \
```

Comment:
Why is this change necessary?

Reply:
Oh, that slipped into this PR and was not supposed to be here. hlint does not know the `--utf8` parameter (any more?).

However, I will revert this commit as it has nothing to todo with _this_ PR.

Reply:
This has been removed in hlint 2.0:
```
2.0, released 2017-04-06
    [...]
    Remove the --encoding/--utf8 options, always use UTF8
    [...]
```